app/models/template.rb

```diff
@@ -82,8 +82,12 @@ def import_without_save(text, options = {})
     self.snippet = !!@importing_metadata[:snippet]
     self.locked = options[:lock] unless options[:lock].nil?
 
-    import_locations(options)
-    import_organizations(options)
+    if options[:taxonomy_params].empty?
```

Comment:
what if I want to only override organizations? could we split this based on incoming params?